src/foam/nanos/app/LegalMenu.js

```diff
@@ -33,6 +33,8 @@ foam.CLASS({
             handler: {
               class: 'foam.nanos.menu.LinkMenu',
               link$: this.slot(function(appConfig$termsAndCondLink) {
+                if ( appConfig$termsAndCondLink.includes("https") )
```

Comment:
```suggestion
                if ( appConfig$termsAndCondLink.includes("http") )
```

Reply:
Problem reading review. It should be https, right?